windows/ReactNativeWebView/ReactWebView2.cpp

```diff
@@ -106,6 +136,35 @@ namespace winrt::ReactNativeWebView::implementation {
         }
     }
 
+    void ReactWebView2::HandleMessageFromJS(winrt::hstring const& message) {
+        winrt::JsonObject jsonObject;
+        if (winrt::JsonObject::TryParse(message, jsonObject) && jsonObject.HasKey(L"type")) {
```

Comment:
you'll want to make sure that the json value kind is string (same in other places).

One pattern you can use instead of HasKey:

```cpp
if (auto v = jsonObject.Lookup(L"type"); v && v.ValueType() == JsonValueType::String) {
  auto type = v.GetString();
  ...
```

Reply:
This section was just copied from the original WebView code. Seems like a good change to make though.